README.md

```diff
@@ -11,6 +11,12 @@ This project contains the Typescript/Javascript bindings for the discovery servi
 npm install @adastradev/serverless-discovery-sdk
 ```
 ## Usage
+
+## Deployment Guide
+This project uses a Continuous Deployment pipeline to deploy directly into an AWS account. If desired, forking the repository to a free Bitbucket account is the easiest way to run the pipeline. The pipeline depends on the following environment variables:
```

Comment:
It would be more accurate to say that we use Continuous Deployment to release to NPM. It's perfectly fine to use bitbucket pipelines for this project, though we could consider Travis, CircleCI, etc if needed in the future.